### PR TITLE
Daily PR June 19

### PR DESCRIPTION
**Pull request by pe7r**
Work with e2e test. 

https://app.asana.com/0/1120909666616459/1127717445239805

### CODE REVIEW

File: src/Components/Header/Header.js [hunk 1]
```diff
@@ -57,7 +57,10 @@ export default class Header extends Component {
                         <NavLink to="/" className="header__navlink"> Contemporary </NavLink>
                         <MoreStyles />
                         <NavLink to="/" className="header__navlink"> Sale </NavLink>
-                        <NavLink to="/products" className="header__navlink" onClick={() => onFilterProducts([])}> Shop All Rugs </NavLink>
+                        <NavLink to="/products"
+                                 className="header__navlink"
+                                 onClick={() => onFilterProducts([])}
+                                 name="shopAllRugs"> Shop All Rugs </NavLink>
```
Comment (ruslanpalagin) [inline src/Components/Header/Header.js]: Name is usually used for other purposes. It's better to not abuse "name" attribute. You can use an existing class name.

File: src/App.test.js [hunk 2]
```diff
@@ -7,3 +7,48 @@ it('renders without crashing', () => {
   ReactDOM.render(<App />, div);
   ReactDOM.unmountComponentAtNode(div);
 });
+
+
+const puppeteer = require('puppeteer');
+
+describe('Products title', () => {
+  test('Title loads correctly', async () => {
```
Comment (ruslanpalagin) [inline src/App.test.js]: Nice) Now you can split it into 2 test cases.
Hint: You can move all preparation staff into some helper

File: src/Components/Header/Header.js (hunk 1 above)
Comment (pe7r) [reply src/Components/Header/Header.js]: Done.

File: src/App.test.js (hunk 2 above)
Comment (pe7r) [reply src/App.test.js]: Done. Will improve.

**Pull request merged**